src/main/scala/com/monsanto/arch/cloudformation/model/resource/DynamoDB.scala

```diff
@@ -21,7 +21,8 @@ case class `AWS::DynamoDB::Table`(
                                    TableName: Option[Token[String]],
                                    override val Condition: Option[ConditionRef] = None,
                                    override val DeletionPolicy: Option[DeletionPolicy] = None,
-                                   override val DependsOn: Option[Seq[String]] = None
+                                   override val DependsOn: Option[Seq[String]] = None,
+                                   TimeToLiveSpecification: Option[TimeToLiveSpecification] = None
```

Comment:
Minor style point -- we usually put the parameters directly related to the resource prior to condition, deletionpolicy, and dependson.  Mind moving it up?  It should remain backwards compatible.

Reply:
Updated.